### PR TITLE
New tool: python2-libvhdi-python

### PR DESCRIPTION
**Pull request by osospeed**

### CODE REVIEW

File: packages/python2-libvhdi-python/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgver=20181227
+pkgrel=1
+pkgdesc='Library and tools to access the Virtual Hard Disk (VHD) image format.'
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/libyal/libvhdi'
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
+license=('LGPLv3+')
+depends=('python2')
+makedepends=('python2' 'python2-setuptools' 'gcc')
+source=("https://pypi.python.org/packages/source/l/libvhdi-python/libvhdi-python-$pkgver.tar.gz")
```
Comment (FFY00) [inline packages/python2-libvhdi-python/PKGBUILD]: You could use the github tarballs but that's not really a dealbreaker.

File: packages/python2-libvhdi-python/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# See COPYING for license details.
+
+pkgname=python2-libvhdi-python
+pkgver=20181227
+pkgrel=1
+pkgdesc='Library and tools to access the Virtual Hard Disk (VHD) image format.'
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/libyal/libvhdi'
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
+license=('LGPLv3+')
```
Comment (FFY00) [inline packages/python2-libvhdi-python/PKGBUILD]: Oh, forgot about this. Should be `LGPL3`.

File: packages/python2-libvhdi-python/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+depends=('python2')
+makedepends=('python2' 'python2-setuptools' 'gcc')
```
Comment (FFY00) [inline packages/python2-libvhdi-python/PKGBUILD]: `python2` shouldn't be here since it's already in `depends`.

**Comment by FFY00** [comment]
This goes for all your other pull requests.

File: packages/python2-libvhdi-python/PKGBUILD (hunk 3 above)
Comment (noptrix) [reply packages/python2-libvhdi-python/PKGBUILD]: ACK

**Pull request merged**